prac_05/emails.py

```diff
@@ -0,0 +1,32 @@
+"""
+Emails
+Estimate: 30 minutes
+Actual: 50 minutes
+"""
+
+email = str(input("Email: "))
+email_to_name = {}
+
+while email != "":
+    # decipher user's name
+    sections = email.split('@')
+    if '.' in sections[0]:
+        name = ' '.join(sections[0].split('.'))
+    else:
+        name = str(sections[0])
+
+    # check if user's name is correct
+    print(f"Is your name {name}?")
+    answer = input("(Y/N): ").lower()
+    if answer == 'y' or answer == "":  # if name is correct
```

Comment:
Nice documentation, but in my opinion your code is easy enough to read as is. "Self documenting" is the wise term Lindsay used (I think) in regards to this